### PR TITLE
Added some tags for channel use

### LINKED ISSUE #39: tag: mod development channels
a tag that explains that questions related to mod development ought to be asked in the #mod-dev-{1,2,3} channels and provides a clickable blue rectangular channel link things for each of them because typing "#mod-dev-1 #mod-dev-2 #mod-dev-3" is a pain. It should also instruct to find the least busy channel and state that ongoing conversations ought to not be interrupted if possible.

### PR DESCRIPTION
**Pull request by YTG1234**
Fixes #39 

This one is simple.

### CODE REVIEW

File: tags/faq/moddev.ytag [hunk 1]
```diff
@@ -0,0 +1,8 @@
+type: text
+
+---
+
+You shall use one of <#507982478276034570>, <#721100785936760876> or <#760651800570757150> for Fabric mod development questions.
```
Comment (i509VCB) [inline tags/faq/moddev.ytag:5]: Feels a bit too commanding. Replace shall with should

**Review by i509VCB — CHANGES_REQUESTED** [review]

**Review by i509VCB — APPROVED** [review]
Want to let others have a bit of time here before I move forward

File: tags/faq/moddev.ytag [hunk 2]
```diff
@@ -0,0 +1,8 @@
+type: text
+
+---
+
+You should use one of <#507982478276034570>, <#721100785936760876> or <#760651800570757150> for Fabric mod development questions.
```
Comment (user11681) [inline tags/faq/moddev.ytag:5]:
```suggestion
You should use one of <#507982478276034570>, <#721100785936760876> and <#760651800570757150> for Fabric mod development questions.
```

**Review by user11681 — CHANGES_REQUESTED** [review]

File: tags/faq/moddev.ytag (hunk 2 above)
Comment (i509VCB) [reply tags/faq/moddev.ytag:5]: Yes, but this is now outdated due to channel changes.

**Review by Platymemo — APPROVED** [review]

**Review by Dolphin2Point1 — APPROVED** [review]

**Pull request merged**